src/esp/metadata/attributes/LightLayoutAttributes.h

```diff
@@ -58,19 +58,19 @@ class LightInstanceAttributes : public AbstractAttributes {
    * @brief Get/Set inner cone angle for spotlights.  Should be ignored for
    * other lights
    */
-  void setInnerConeAngle(double innerConeAngle) {
-    setDouble("innerConeAngle", innerConeAngle);
+  void setInnerConeAngle(const Magnum::Rad& innerConeAngle) {
```

Comment:
It's a tiny type with trivial copy, so just

```suggestion
  void setInnerConeAngle(Magnum::Rad innerConeAngle) {
```

is faster :) Same below and in `core/Configuration.h` above.